actions/update/action.yaml

```diff
@@ -0,0 +1,157 @@
+name: Update Flux manifests
+description: A GitHub Action for updating Flux manifests in bootstrap repositories
+author: Stefan Prodan
+branding:
+  color: blue
+  icon: command
+inputs:
+  registry:
+    description: "Distribution Container Registry"
+    default: "ghcr.io/controlplaneio-fluxcd"
+    required: true
```

Comment:
do we need this to be `true` when we have a `default`? Same question for `variant` and `components`.

Reply:
Yes, as you can delete the content from the textbox and the action will fail.